projects/1.20-fabric/assets/observable/observable/lang/zh_cn.json

```
@@ -0,0 +1,34 @@
+{
+  "screen.observable.profile": "轮廓",
+  "screen.observable.results": "统计",
+  "key.observable.profile": "显示",
+  "category.observable.keybinds": "Observable",
+  "text.observable.profile_tps": "TPS占用显示",
+  "text.observable.profile_fps": "FPS占用显示",
+  "text.observable.sampler": "加载采样器的配置文件（增加性能消耗）",
+  "text.observable.unimplemented": "未完成！",
+  "text.observable.unavailable": "服务器上未安装Observable",
+  "text.observable.no_permissions": "你没有权限运行",
+  "text.observable.error": "执行操作时出错：%s",
+  "text.observable.report": "请在github.com/tasgon/observable/issues上报告错误",
+  "text.observable.entity_not_found": "在%s中找不到实体，他可能死了或者被移走了？",
+  "text.observable.get_field": "输入配置文件URL或哈希……",
+  "text.observable.get_btn": "Get!",
+  "text.observable.overlay": "显示",
+  "text.observable.docs": "Wiki",
+  "text.observable.discord": "Discord",
+  "text.observable.donate": "Donate",
+  "text.observable.profile_started": "Observable 正在分析 %d 秒之内的游戏...",
+  "text.observable.profile_uploaded": "分析结果已上传到 %s",
+  "text.observable.upload_failed": "无法上传分析结果",
+  "text.observable.profile_saved": "分析结果已保存到 %s",
+  "text.observable.after_save": "在 %s 处打开此文件",
+  "text.observable.suggest": "您的服务器似乎运行得有点慢（%s tps） ，%s 查找滞后源。",
```

```suggestion
  "text.observable.suggest": "你的服务器似乎运行得有点慢（%s tps）。%s以查找卡顿源。",
```
这里在游戏内是怎么显示的

---
若使用翻译辅助，请注意译后编辑修正
直接提交未经修正的版本会被拒收